lua/gatekeeperc.lua

```diff
@@ -14,6 +14,7 @@ struct net_config {
 	uint32_t		num_ports;
 	uint32_t		numa_nodes;
 	struct rte_mempool 	**gatekeeper_pktmbuf_pool;
+	uint16_t		queues[128];
```

Comment:
Can't we use the define here? I know that we'll still have to repeat the number in the define, but if the define is used again, we'll have avoided a duplicate. Moreover, knowing the identifier here helps to understand the limit.